index.html

```diff
@@ -2,18 +2,154 @@
 <hmtl>
   <head>
     <title>YouTube</title>
+    <link rel="stylesheet" href="styles/index.css" />
   </head>
   <body>
     <!-- Navbar here-->
     <header>
-      <nav class="navbar"></nav>
+      <nav class="navbar">
+        <div class="grid-item grid-item1">
+          <img class="logo" src="img/Youtubelogo.png" alt="logo" />
+        </div>
+
+        <div class="grid-item grid-item2">
+          <input type="text" name="search" placeholder="Search" />
+          <button type="button"><img src="img/search.png" /></button>
+        </div>
+
+        <div class="grid-item grid-item3">
+          <a href="https://www.youtube.com/" class="nav_links"
+            ><img src="img/notification.png" />
+          </a>
+          <a href="#" class="nav_links"><img src="img/camera.png" /></a>
+          <a href="#" class="nav_links"><img src="img/menu.png" /></a>
+        </div>
+      </nav>
     </header>
 
     <main class="main-content">
       <!-- Main video here -->
       <section class="video_big"></section>
       <!-- Side bar here -->
-      <section class="sidebar"></section>
+      <section class="sidebar">
+      <div class="thumbnail">
+          <div class="video">
+            <iframe
+              width="168"
+              height="94"
+              src="https://www.youtube.com/embed/oIhNoeKFwbE"
+              frameborder="0"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
+          <div class="thumbnail-text">
+            <h1 style="color: white;">Hello, I am this video's text</h1>
+            <p style="color: white;">John Doe</p>
+          </div>
```

Comment:
## No inline styling

Using Inline Styles is probably one of the worst CSS practices that many web designers and developers are still doing to this day. It transgresses the main goal of why we use CSS – which is to separate style from the HTML structure. This breaks the specificity of web pages

- [ ] Remove all `style` inline you have in the HTML file

**Note:** Remove all inline style rules in the entire HTML page

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_